src/main/app/src/app/admin/parameter-edit/smart-documents/smart-documents-tree.component.ts

```diff
@@ -183,7 +183,10 @@ export class SmartDocumentsTreeComponent {
     ],
     queryFn: () =>
       firstValueFrom(
-        this.smartDocumentsService.getTemplatesMapping(this.zaaktypeUuid),
+        this.smartDocumentsService.getTemplatesMapping(
+          this.zaaktypeUuid,
+          "TODO",
```

Comment:
@felixcicatt Where do we take this from? Perhaps this can be changed with a follow-up task?

Reply:
Hmm yeah. Better to fix this in this PR if you still can I think?

Reply:
I think we will need to add a UI element so you can first select a informatieobjecttype before the tree view is displayed. If you don't want to break anything in this PR without adding new UI we could fill in the id of the first document type here I guess?

Reply:
Using null UUID now. In this way we have a valid UUID for the calls.
As the backend deletes & then inserts the new mapping is should not be a problem later